examples/plugin/wasi-logging/README.md

```diff
@@ -58,32 +62,32 @@ wasmedge target/wasm32-wasi/debug/wasi-logging-example.wasm
 
 This example should run successfully and print out the log as follow.
 
+```sh
+[2024-05-20 17:47:03.030] [wasi_logging_stdout] [info] ===================================
```

Comment:
Is it necessary to show `[wasi_logging_stdout]` in log messages?

Reply:
@q82419 Maybe we can simply them into `[wasi_logging]`. The users will know stderr or stdout when they retrieve the log.

Reply:
Oh, due to the spdlog limitation, we have to keep the tag in a different name to avoid conflict.

Reply:
Maybe we should figure out a way to remove these tags directly.

Reply:
Removed the logger tag.